.github/workflows/windows_vs2019_debug.yml

```diff
@@ -65,7 +69,8 @@ jobs:
       runner: 'aks-win-4-cores-8gb'
       affected-components: ${{ needs.smart_ci.outputs.affected_components }}
       os: 'windows_2019'
-
+      gtest-filter: '-TransformationTestsF.RemoveIfDanglingParametersFromBodiesAndInputs:-TokenizeMHASnippetsTests.smoke_Snippets_MHA3D_SplitM_withMul' # Issue 155216
```

Comment:
can they be disabled via C++ code itself? 
Hardcoding filters in CI service is bad idea.

Reply:
disabled tests themselves